src/rules/converters/jsdoc-format.ts

```diff
@@ -0,0 +1,18 @@
+import { RuleConverter, ConvertedRuleChanges } from "../converter";
+
+export const convertJSDocFormat: RuleConverter = () => {
+    const ruleNames: string[] = [
```

Comment:
Nit:

```suggestion
    const ruleNames = [
```

Looks like a `no-inferrable-types` violation here.

Reply:
I do feel like ESLint is not working at all at my forked project, because I didn't got the warning message. Gonna check again 👍